jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -185,6 +185,13 @@ def _on_link_type_updated(self, msg=None):
         # toggle visibility as necessary
         self._on_is_active_changed()
 
+        # When fp(s) added via API before WCS link. Overlays visibility & is_active
+        # can be True, but only last fp will display. This ensures all fp(s) display

```

Comment:
One nitpick, please write out "footprint" in the comments rather than using potentially opaque abbreviations, for future readers' sake.

Reply:
I totally agree, change made!